### ISSUE TITLE
Vesicular eruption

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6779]
**Preferred term label:**
Vesicular eruption

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of acute-onset skin rash characterized by multiple vesicles, which are circumscribed, fluid-containing, epidermal elevation generally considered less than 10mm in diameter at the widest point. The flud contained in a vesicle may be clear, serous or haemorrhagic.
PMID: 33008676

Children
Monomorphic vesicular eruption (This is the COVID-19 phenotype)
A type of vesicular eruption in which the vesicles are at same stages, unlike polymorphic vesicles in chickenpox.
PMID:32348545 

Polymorphic vesicular eruption
A type of vesicular eruption in which the vesicles are at different stages

**Parent term (use hpo.jax.org/app)**
Skin vesicle HP:0200037
Skin rash

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
